TAGradingServer/account/admin-latedays-exceptions.php

```diff
@@ -0,0 +1,752 @@
+<?php
+
+//Author: Peter Bailie, Systems Programmer, RPI Computer Science, July 2016
+
+/* MAIN ===================================================================== */
+
+include "../header.php";
+
+check_administrator();
```

Comment:
I can remove the ```if ($user_is_administrator)``` condition, but it does exist in a few other "admin" php pages.